### ISSUE TITLE
[Accessibility] Section element in fd-wizard-content is missing accessible name

### ISSUE THREAD

**Issue by pawelkarwat** [SAP/fundamental-ngx#13540]
### Describe your proposal

Expected behavior:
Any element with role="region" must have a descriptive accessible name (via aria-label, aria-labelledby, or a native h1-h6 heading inside. This helps screen readers announce the region meaningfully and improves navigability for assistive technology users.

### Which versions of Angular and Fundamental Library for Angular are affected?

Angular: 20, Fundamental Library: 0.57.3

### Steps to Reproduce (for bugs)

1. Go to wizard component

<img width="2550" height="1404" alt="Image" src="https://github.com/user-attachments/assets/77b77abc-3cf4-4c3e-8f48-53119ca2e82b" />

### Declaration

- [x] I'm not disclosing any internal or sensitive information.
